custom_components/tuya_local/devices/tmwf02_fan.yaml

```diff
@@ -12,6 +12,21 @@ primary_entity:
       range:
         min: 1
         max: 100
+    - id: 3
+      type: string
+      name: fan_level
+      mapping:
```

Comment:
No need for the mapping if it is hidden.  The mapping is only to 1) map to different values, 2) to make the list of possible values known to the UI.  If it is hidden, there is no need for either of these.

Reply:
Ok, I delete it.